### PR TITLE
feat(annotations): add cta for span annotations

### LINKED ISSUE #4142: [annotations][ui] call to actions in the feedback tables on how to add annotations
As a user, if I click on the feedback and see empty tables I might want to know how to add feedback / annotations. Add a button called "add annotations" that both explains annotations briefly but also links out to documentation so that the user can learn more.

### PR DESCRIPTION
**Pull request by Parker-Stafford**
resolves #4142 

- adds cta to both annotate dialog and feedback tab on span details - only displays when there are no annotations of either type (human / llm)

image

Screenshot 2024-08-07 at 11 52 11 AM

### CODE REVIEW

File: app/src/pages/trace/SpanAnnotationsEmpty.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      </Dialog>
+    );
+  };
+  return (
+    <View padding={"size-200"}>
+      <Flex direction="column" gap="size-100" alignItems="center">
+        <EmptyGraphic graphicKey="documents" />
+        <Text>No annotations for this span</Text>
+        <Button variant="default" onClick={onGettingStartedClick}>
+          Add Annotations
```
Comment (mikeldking) [inline app/src/pages/trace/SpanAnnotationsEmpty.tsx:44]:
```suggestion
          How to Annotate
```

File: app/src/pages/trace/SpanAnnotationsEmpty.tsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        </View>
+      </Dialog>
+    );
+  };
+  return (
+    <View padding={"size-200"}>
+      <Flex direction="column" gap="size-100" alignItems="center">
+        <EmptyGraphic graphicKey="documents" />
+        <Text>No annotations for this span</Text>
+        <Button variant="default" onClick={onGettingStartedClick}>
```
Comment (mikeldking) [inline app/src/pages/trace/SpanAnnotationsEmpty.tsx:43]: maybe add a pencil icon

Comment (mikeldking) [inline app/src/pages/trace/SpanAnnotationsEmpty.tsx:43]:
```suggestion
        <Button variant="default" size="compact" onClick={onGettingStartedClick}>
```

File: app/src/pages/trace/EditSpanAnnotationsDialog.tsx [hunk 3]
```diff
@@ -227,9 +227,7 @@ function SpanAnnotationsList(props: {
   const hasAnnotations = annotations.length > 0;
   return (
     <div>
-      {!hasAnnotations && (
-        <Empty graphicKey="documents" message="No annotations for this span" />
-      )}
+      {!hasAnnotations && <SpanAnnotationsEmpty />}
```
Comment (mikeldking) [inline app/src/pages/trace/EditSpanAnnotationsDialog.tsx:230]: I'm not sure this makes sense here. You can add the annotations in this UI. I would remove here.

**Review by mikeldking — APPROVED** [review]
Left some comments

File: app/src/pages/trace/EditSpanAnnotationsDialog.tsx (hunk 3 above)
Comment (Parker-Stafford) [reply app/src/pages/trace/EditSpanAnnotationsDialog.tsx:230]: ahh good point will remove

**Pull request merged**